suggestions/barcode_scanners.json

```diff
@@ -4,5 +4,24 @@
         "label": "Binary Eye",
         "source": "fg",
         "repo": "https://github.com/markusfisch/BinaryEye"
+    },
+    {
+        "id": "com.example.barcodescanner",
```

Comment:
Looks abandoned.

Reply:
Abandoned, still connects to Sentry. Upstream [PR](https://github.com/wewewe718/QrAndBarcodeScanner/pull/151) for that still open & no response from the Owner, which is why [F-Droid team fixed it](https://gitlab.com/fdroid/fdroiddata/-/issues/2846) from their side.

Reply:
The app doesn't offer anything better than the other two candidates anyway.